sotodlib/io/load.py

```diff
@@ -587,6 +588,7 @@ def hstack_into(dest, src_arrays):
     {
         'pipe-s0001': load_observation,
         'toast3-hdf': load_toast_h5_obs,
+        'obs-book': load_obs_book,
```

Comment:
could also just stick a line in `load_book` like this:

https://github.com/simonsobs/sotodlib/blob/29c83f97d1928dba15ff7c29453bfaca16ec74be/sotodlib/io/load_smurf.py#L2479 

It's not clear to me why we need all these here and/or if I should have the level 2 one here.

Reply:
I think if one wants to draw a line between what could be here and what should instead only be added on submodule import, it is whether the submodule has additional non-standard / weird dependencies (e.g. moby2, sqlalchemy (?)).  I don't mind moving this and the toast_h5 one out ... I think I mostly was just being quick and forgot to clean up.